index-source.txt

```diff
@@ -80,7 +80,7 @@ path: stylesheet-extra.include
 
 		* Glyphs representing character are shown as: GLYPH{&#x239C2;}.
 
-		* Unicode character encodings are shown as: UNI{U+003d}.
+		* Unicode code points are shown as: UNI{U+003D}.
```

Comment:
```suggestion
		* Unicode [=code points=] are shown as: UNI{U+003D}.
```

It would be useful to add a link to the definition.

Reply:
Done! Thanks @xfq .